docs/dispacher-install.md

```diff
@@ -30,6 +30,14 @@ mvn install -Dmaven.test.skip=true
 
 Edit the files:
 - [Dispatcher configuration file](/config/dispatcher.conf) to allow its communication with the SAPS Catalog and Permanent Storage.
+- The ```$openstack_object_store_service_key``` argument in [Dispatcher configuration file](/config/dispatcher.conf) used to access Object Store must be configured using the command ```swift post -m “Temp-Url-Key:$openstack_object_store_service_key”``` passing as other information as parameters. Use a random 8 chars string and run the following command to configure:
+
+```bash
+swift post -m "Temp-URL-Key:$openstack_object_store_service_key" --os-auth-url $openstack_identity_service_api_url --auth-version 3 --os-user-id $openstack_user_id --os-password $openstack_user_password --os-project-id $openstack_project_id
+```
+
+Note: ```-auth-version``` is version in operation of the Openstack Identity Service API
```

Comment:
replace by

Note: ```-auth-version``` is the version of the deployed Openstack Identity Service API

Reply:
Done